databricks/sdk/core.py

```diff
@@ -1,1040 +1,24 @@
-import abc
-import base64
-import configparser
-import copy
-import functools
-import io
 import json
 import logging
-import os
-import pathlib
-import platform
 import re
-import subprocess
-import sys
 import urllib.parse
-from datetime import datetime, timedelta
+from datetime import timedelta
 from json import JSONDecodeError
 from types import TracebackType
-from typing import (Any, BinaryIO, Callable, Dict, Iterable, Iterator, List,
-                    Optional, Type, Union)
+from typing import (Any, BinaryIO, Dict, Iterable, Iterator, List, Optional,
+                    Type, Union)
 
-import google.auth
 import requests
-from google.auth import impersonated_credentials
-from google.auth.transport.requests import Request
-from google.oauth2 import service_account
 from requests.adapters import HTTPAdapter
 
-from .azure import (ARM_DATABRICKS_RESOURCE_ID, ENVIRONMENTS, AzureEnvironment,
-                    add_sp_management_token, add_workspace_id_header)
+from .config import Config
 from .errors import DatabricksError, error_mapper
-from .oauth import (ClientCredentials, OAuthClient, OidcEndpoints, Refreshable,
-                    Token, TokenCache, TokenSource)
 from .retries import retried
-from .version import __version__
 
 __all__ = ['Config', 'DatabricksError']
```

Comment:
i'm relying on `credentials_provider` and couple of other classes from `core`. don't break it. Please verify that https://github.com/databrickslabs/ucx still works before this PR is merged. If you make a PR to accommodate these changes - also fine

Reply:
Just added imports in `core.py` to make sure all credentials_provider.py and config.py classes are reexported. Manually tested and was able to import those definitions after installing.